backends/p4tools/scripts/tools/iwyu_mappings/gcc.libc.imp

```diff
@@ -0,0 +1,211 @@
+# Mappings for GNU libc
```

Comment:
what are these imp files? Do they need to be in the repository?

Reply:
They are mapping files that are used by IWYU. They are quite useful because they help with generating the right includes when using the tool. Without them, IWYU will not be usable.

Reply:
did you write them by hand?  if they are generated by a tool we should use the tool to generate the files.
or perhaps they come from a repository?

Reply:
https://github.com/include-what-you-use/include-what-you-use
They are handpicked from here. We could potentially link against this module and run the tool from there. 
`p4c.imp` is written by us.

Reply:
@fruffy It might be useful to add a `Makefile` that would make it easier to update these files in the future:

```Makefile
all: gcc.libc.imp libcxx.imp stl.c.headers.imp stl.c.headers.imp boost-1.75-all.imp

gcc.libc.imp:
	curl -s https://raw.githubusercontent.com/include-what-you-use/include-what-you-use/master/gcc.libc.imp -o $@

libcxx.imp:
	curl -s https://raw.githubusercontent.com/include-what-you-use/include-what-you-use/master/libcxx.imp -o $@

stl.c.headers.imp:
	curl -s https://raw.githubusercontent.com/include-what-you-use/include-what-you-use/master/stl.c.headers.imp -o $@

boost-all.imp:
	curl -s https://raw.githubusercontent.com/include-what-you-use/include-what-you-use/master/boost-all.imp -o $@

boost-1.75-all.imp:
	curl -s https://raw.githubusercontent.com/include-what-you-use/include-what-you-use/master/boost-1.75-all.imp -o $@
```

It would be useful to explain how these files are used in `README.md`.